capstone-frontend/src/pages/Home.jsx

```diff
@@ -9,14 +9,37 @@ function Home() {
     const params = useParams();
     const variable = params.otherid
 
+    async function getBorderData() {
+        const response = await (async () => {
+            return fetch('http://localhost:5000/borderdata');
+        }) (); 
+        const data = await response.json();
+        console.log(data)
+        return data;
+    }
+
     useEffect(() => {
```

Comment:
I'd simplify all of this to 
useEffect(()=>{
const response = await fetch('...');
const jsonData= await response.json();
setReadyData(jsonData)
},[setReadyData]);